src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -403,6 +417,7 @@ def export_model(self, model_path, mask_path=None, onnx_path=None, input_shape=N
             _logger.info('Model in onnx with input shape %s saved to %s', input_data.shape, onnx_path)
 
         self._wrap_model()
+        return mask_dict, self.bound_model.state_dict()
```

Comment:
update docstring to add "Returns"

Reply:
The change is not needed any more, removed the change.